### PR TITLE
Refactor plaintextTitle to utilise displayName

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?

We have inconsistent display values scattered through the author application. In some places, we display titles, in other places we display labels and in other places we have inconsistent unnamed or untitled placeholders.

This technical improvement makes it the responsibility of the API to provide the most relevant displayable value.

More info can be found here: https://github.com/ONSdigital/eq-author/wiki/Display-names

### How to review

- Run tests

### Dependencies 

- [x] https://github.com/ONSdigital/eq-author-graphql-schema/pull/63
- [x] https://github.com/ONSdigital/eq-author-api/pull/122

### CODE REVIEW

File: cypress/integration/authenticated/routing_spec.js [hunk 1]
```diff
@@ -292,8 +281,7 @@ describe("Routing", () => {
 
     cy.get(testId("routing-rule"));
 
-    cy
-      .get(testId("result-selector"))
+    cy.get(testId("result-selector"))
```
Comment (tgandrews) [inline cypress/integration/authenticated/routing_spec.js]: We should run prettier on the whole codebase as these sorts of changes are a little noisy in every PR.

File: src/components/QuestionnaireMenu/index.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const PageMenu = ({ pages, sectionNumber, menuZIndex, ...otherProps }) => (
   <Dropdown maxWidth={"25em"}>
     <MenuList maxHeight={`${MAX_HEIGHT}em`}>
       {map(pages, (page, i) => {
-        const title = page.plaintextTitle;
+        const title = page.displayName;
```
Comment (tgandrews) [inline src/components/QuestionnaireMenu/index.js]: This file also has defaults on lines 31 and 51 that I think can be removed

File: cypress/integration/authenticated/routing_spec.js (hunk 1 above)
Comment (adammahmood) [reply cypress/integration/authenticated/routing_spec.js]: @tgandrews Yea I agree.  I will do it shortly when I'm inbetween tasks.

**Comment by tgandrews** [comment]
PR looks good just one issue discussed on slack with the piping menu answer name not being updated.

**Comment by adammahmood** [comment]
@tgandrews Fixed the answer not updating in the piping menu

**Comment by samiwel** [comment]
[Introduce displayName](https://trello.com/c/zUaulBzP/607-introduce-displayname)

File: src/containers/enhancers/withDeleteSection.js [hunk 3]
```diff
@@ -68,7 +68,9 @@ export const deleteUpdater = (questionnaireId, sectionId) => (
 };
 
 export const displayToast = (ownProps, questionnaire) => {
-  const { match: { params } } = ownProps;
+  const {
```
Comment (hamishtaplin) [inline src/containers/enhancers/withDeleteSection.js]: Any idea why this is being reformatted like this? Original formatting was much nicer

Comment (adammahmood) [reply src/containers/enhancers/withDeleteSection.js]: Think it's related to the Prettier upgrade.  I'll run on the entire codebase and raise a separate PR.

File: src/components/RichTextEditor/entities/PipedValue.js [hunk 4]
```diff
@@ -78,7 +78,7 @@ export const insertPipedValue = (answer, contentState, selection) => {
   );
 
   const entityKey = newContent.getLastCreatedEntityKey();
-  const text = answer.label || "Piped Value";
+  const text = answer.displayName;
```
Comment (hamishtaplin) [inline src/components/RichTextEditor/entities/PipedValue.js]: We should make this change everywhere so there's no display logic around display of labels/titles and everything defers to the response from the API.

File: src/components/RichTextEditor/entities/PipedValue.js [hunk 5]
```diff
@@ -64,7 +64,7 @@ export const replacePipedValues = labels => (
 ) => {
   const text = labels.hasOwnProperty(entity.data.id)
     ? labels[entity.data.id]
-    : "Deleted Piped Value";
+    : `Untitled Answer`;
```
Comment (hamishtaplin) [inline src/components/RichTextEditor/entities/PipedValue.js]: Is this right? I thought we said leave deleted answers as-is.

**Pull request merged**